src/models/countries.js

```diff
@@ -0,0 +1,46 @@
+module.exports = (sequelize, DataTypes) => {
+  const Country = sequelize.define("Country", {
+    id: {
```

Comment:
in models, you do not need to include the id field, remove it

Reply:
corrected